node/builder.go

```diff
@@ -384,6 +384,9 @@ func Test() Option {
 		Unset(new(*peermgr.PeerMgr)),
 		Override(new(beacon.Schedule), testing.RandomBeacon),
 		Override(new(*storageadapter.DealPublisher), storageadapter.NewDealPublisher(nil, storageadapter.PublishMsgConfig{})),
+		// use the testing bundles
+		Unset(new(dtypes.BuiltinActorsLoaded)),
+		Override(new(dtypes.BuiltinActorsLoaded), modules.LoadBuiltinActorsTesting),
```

Comment:
Do we want this in the real thing?

Reply:
Yes? It does the magic to make us use the testing builtin actors bundles for our itests.

Reply:
But do we want this in non-test nodes?

Reply:
I don't understand this question. What's a non-test node? We just need this to happen when we run itests (and never else), and I thought that's what this does, but idkkk.

Reply:
Oh this is the `Test` func thing, I thought this was part of the default node options